### PR TITLE
Leaderboard: hit the API

### PR DESCRIPTION
**Pull request by AmaliaGM**
## Project requirements

- Create a new game with the name of your choice by using the API.
- Implement the "Refresh" button (receiving data from the API and parsing the JSON).
- Implement the form "Submit" button (sending data to the API).
- Use arrow functions instead of the function keyword.
- Use async and await JavaScript features to consume the API.
- No styling is required.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,19 +1,52 @@
...
+      score,
+    }),
+  })
+    .then((response) => response.json());
+};
+
 // ADD NAME AND SCORES
-export default function addElements() {
-  const name = document.createElement('li');
+function addElements() {
```
Comment (KDlamini) [inline src/index.js]: Please use an `arrow function` instead of the `function` keyword.

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @AmaliaGM 🙋‍♀️,  

Great work so far! 💯 
There are some issues that you still need to work on to improve your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 
✔️  Use of `async` and `await` with the `fetch` method
✔️  POST and GET API endpoints used correctly
✔️  App runs as expected
✔️  Good PR title and description
✔️  No linter errors

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mukhammadrizooff — APPROVED** [review]
Hi @AmaliaGM, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️
- Good implementations ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AmaliaGM** [comment]
@mukhammadrizooff Thanks for your review!

**Pull request merged**